### ISSUE TITLE
Brewtarget has two separate bug trackers

### ISSUE THREAD

**Issue by matty0ung** [Brewtarget/brewtarget#493]
AIUI, https://github.com/Brewtarget/brewtarget/issues is now the official bug/feature list.  However, https://bugs.launchpad.net/brewtarget is where new users are likely to end up.  (You start from the http://www.brewtarget.org/ home page, click 'Help' to get to https://groups.google.com/g/brewtarget-help and then click Bug Tracker.)  If Launchpad is now deprecated can we mark it as such, merge its bugs into GitHub and fix the links to point to GitHub?  Happy to help with any of the busywork in doing this.

**Comment by matty0ung** [comment]
While we're at it, we could fix the other broken link at https://groups.google.com/g/brewtarget-help: 'Source Code Repository' points to http://www.gitorious.org/brewtarget instead of https://github.com/Brewtarget/brewtarget

**Comment by mattiasmaahl** [comment]
I don't know who has access to both the web site and the googlegroup, my guess would be @rocketman768  or @mikfire. maybe a general overhaul would be in order?

**Comment by matty0ung** [comment]
Yes, at very least if we could fix the broken and out-of-date links mentioned in https://github.com/Brewtarget/brewtarget/wiki/Where-to-Find-Things that would be helpful to new users.  Agree it's most likely @rocketman768 and @mikfire who have the necessary access.

**Comment by mikfire** [comment]
I think you are terribly optimistic to name me in that. I honestly have no idea if I have access to brewtarget.org. 

I would have loved to shut down the launchpad site entirely, except that was where we actually handled the internationalization work -- rocketman thought the work flow on launchpad was better for this task.

**Comment by mattiasmaahl** [comment]
Ok, I cannot say what system is best, but I used the QT Linguist software when I translated the Swedish translation last. Then its a matter of updating the files here with a PR, but it may be easier for people who is not used to the github process to use Launchpad web-gui for that task as it doesn't need any additional software installed. 

Question is though how we want to go forward from now on? If we choose to continue with Launchpad we need access to it, I can handle the translation integrations if need be, but I need access for that.

As for the web site, maybe you can try to get hold of rocketman regarding access? It has not been updated in quite some time.

The Wiki on Github we can update as much as we agree on. :) removing the old links is a good start I believe and then tackle the other sites at a later point?

**Comment by matty0ung** [comment]
I will ask rocketman about access to brewtarget.org.

**Comment by matty0ung** [comment]
AIUI there are projects that use Launchpad for translations and Github for sourcecode, wiki.  UI on Launchpad for bugs etc looks a bit clunky to me, but its translation management (for which there seems no equivalent on Github) looks quite a bit nicer than editing the .pr files directly. I'm sure we can get admin or similar access to Launchpad from @rocketman768.

**Comment by matty0ung** [comment]
I'm going to close this issue now.  I didn't ever get a response from @rocketman768 on this, so I think we just have to live with the fact that the out-of-date sites linger on in their zombie-like state, and make sure wherever possible that we point folks at the up-to-date places to get the software, find out more about it, raise issues, etc.
